app/scripts/lib/setupSentry.test.js

```diff
@@ -1,4 +1,5 @@
 import { rewriteReport, removeUrlsFromBreadCrumb } from './setupSentry';
+import * as setupSentry from './setupSentry';
```

Comment:
S: Remove above `rewriteReport, removeUrlsFromBreadCrumb` import if you import `*` eventually

Reply:
Good idea ! this is addressed in 32f447e71d540ffef7f23412b5a6819de40ae277